client/src/components/Register.scss

```diff
@@ -0,0 +1,18 @@
+.mfa-register-confirmation {
+  text-align: center;
+  margin: 3em auto 3em;
+
+  &__icon {
+    line-height: 1;
+    color: $color-green;
+    font-size: 76px;
```

Comment:
There's a couple of spots where variables could be used here

Reply:
I moved the `margin-bottom` from the title to a variable and style on `mfa.scss`. I'm not sure if there's value in assigning variables to the other bits? Maybe the font-size on the big tick could be used for another screen. The margin stuff here is provided because this screen is pretty unique in it's spacing - I didn't see a screen that had similar spacing.

I'm happy to remove all the special snowflake spacing if you think we should be consistent?

Reply:
Continual improvement over perfection...